### ISSUE TITLE
Tree Table: Expand and Collapse row is not focusable

### ISSUE THREAD

**Issue by lakha26** [SAP/ui5-webcomponents-react#7147]
### Describe the bug

Hi, the expand/collapse button in the Tree Table is not focusable using the Tab key. I want to add aria-expanded for accessibility so that when we press the Spacebar, it announces 'expanded,' and when pressed again, it announces 'collapsed.

### Reproduction steps

Use tab to navigate to expand/collapse button.

### Expected Behaviour

Expand and Collapse button in should be focusable via Tab.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/a00add3b-134d-45ab-be75-3fd6266c56b5)

![Image](https://github.com/user-attachments/assets/077607bc-f31d-4cc5-bf39-2f1bc5955c32)

![Image](https://github.com/user-attachments/assets/98996b97-2ddb-46bc-ae72-cb952ee04034)

### UI5 Web Components for React Version

1.29.18

### UI5 Web Components Version

1.24.9

### Browser

Chrome

### Operating System

MAC

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @lakha26 

the linked PR will improve this behavior. Please note that the expand button is not included in the tab-chain and can therefore only be focused via pointer interactions. When using the keyboard to navigate the table pressing "Space" when the "expand" cell is focused, the row is expanded/collapsed, which is also announced by screen readers.
